docs/release-notes/3479-new-pytorch-callback-method.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**New Features**
+
+-  API: Add ``on_training_epoch_end`` to PytorchCallback. Add ``epoch_idx`` argument to
+   ``on_training_epoch_start``.
```

Comment:
Can you add to the note that we have backwards compatibility for existing code, so this is not a breaking change?

Reply:
👍